app/workers/concerns/processor.rb

```diff
@@ -0,0 +1,32 @@
+module Processor
```

Comment:
I'm not sure what this class is from the name, it seems like a place to put miscellaneous methods used by workers which could end up becoming untidy.

The idea of "concerns" is also a new one for email-alert-api, generally we've been using the concept of "services" instead which might be a better fit here. For example, I can imagine the courtesy email part being refactored into a service: `CourtesyEmailService.queue(ContentChangeEmailBuilder, content_change: content_change)`

As I understand it, while this is a very nice refactor, it's not specifically part of the code to have multiple immediate email generation workers, so I think it's worth pulling it out into a separate commit (or even a separate PR which could get merged in sooner).

Reply:
It was a bit unclear, I've split those into their own classes and hopefully that'll be clearer

I've also split it into two commits